src/run.ts

```diff
@@ -25,7 +25,7 @@ export default async function run(): Promise<void> {
     const artifacts: Artifact[] = inputs.artifacts
       .flatMap(artifact => {
         const expandedGlob = fastGlob.sync(artifact.path)
-        if (expandedGlob.length > 1) {
+        if (expandedGlob.length > 0) {
           return expandedGlob.map(path => ({
             ...artifact,
             path,
```

Comment:
This makes sense to me- in the case where it _doesn't_ find anything in the glob, we'll store the glob-path as the original path and create an artifact with missing files?

Reply:
Yep!